### ISSUE TITLE
bin/standalone.py with newer Python shows DeprecationWarning on start up

### ISSUE THREAD

**Issue by futatuki** [viewvc/viewvc#361]
On Python 3.11,  standalone.py shows a warning below:
```
$ /usr/local/viewvc/bin/standalone.py
/usr/local/viewvc/bin/standalone.py:63: DeprecationWarning: 'crypt' is deprecated and slated for removal in Python 3.13
  import crypt
```

It is used for supporting --htpasswd-file option.  

Note: [fcrypt](https://pypi.org/project/fcrypt/) is still in PyPI, but I've not checked if it is avalable on Python 3.

**Comment by futatuki** [comment]
As the library reference for [crypt](https://docs.python.org/3.12/library/crypt.html) says, we could use [passlib](https://pypi.org/project/passlib/) for this purpose.  Here is the document of "[passlib.apache - Apache Password Files](https://passlib.readthedocs.io/en/stable/lib/passlib.apache.html)".

**Comment by futatuki** [comment]
The issue was resolved in PR #362.  As it was merged, close this as completed.
